src/test/end-to-end/helpers/caseHelper.js

```diff
@@ -129,7 +131,18 @@ async function printHearingLists(I, eventName, jurisdiction) {
 async function caseTransfer(I, eventName) {
     await I.chooseNextStep(eventName, 3);
     await I.wait(2);
```

Comment:
remove the await for this I.wait() method....

Reply:
Removed the await for this method